extensions/functions_string.yaml

```diff
@@ -17,6 +17,28 @@ scalar_functions:
         variadic:
           min: 1
         return: "string"
+  - name: chr
+    description: >-
+    Returns the ASCII character having the binary equivalent to input.
+    If n is larger than 256 the result is equivalent to chr(n % 256).
+    If n is negative value, the result is null.
+    impls:
+    - args:
+      - value: i8
+        name: "input"
+      return: "fixedchar<L1>"
```

Comment:
What is `L1` here?  None of the input arguments are parameterized so I don't think the output can be parameterized.  Do you perhaps want `fixedchar<1>`?  It seems postgres, duckdb, datafusion, and presto all return a variable size string.  Only SQL server returns a fixed size string.  I think returning `string` here would be more consistent with existing implementations.